portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -13,20 +13,37 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
 
+  private List<String> phrases;
+
+   @Override
```

Comment:
nit: looks like there is an extra space before this `@Override` statement.

Reply:
Got it, I'll push the update